target/data.txt

```diff
@@ -0,0 +1,7 @@
+The Course
+Java
+Lesson 6
```

Comment:
Czemu to jest tutaj?

Reply:
+ czemu to jest w commicie:
![image](https://user-images.githubusercontent.com/17989854/134807489-688d0b6d-0b32-4cf0-a8b3-69c758d759a9.png)